sys/d/manager.d

```diff
@@ -1719,16 +1719,18 @@ EOS";
 					[
 						config.build.components.website.diffable
 						? ["all", "verbatim", "pdf"] ~ (
-							makeFullName.readText.indexOf("diffable-intermediaries") >= 0
+							makeFullText.indexOf("dautotest") >= 0
```

Comment:
This doesn't look like it's doing what you meant. `dautotest` already includes `all`, `verbatim` and `pdf`, doesn't it?

Reply:
Oh sorry. I accidentally reverted that bit :/